app/views/feedback/new.html.slim

```diff
@@ -1,21 +1,15 @@
-- content_for(:page_title, t('.header'))
-
-.main-header role="banner"
-  h1= t '.header'
+h1.govuk-heading-xl= t("claims.feedback.header")
 
 .main-section
   .main-content
-    = simple_form_for(resource, url: feedback_path) do |f|
+    = form_for resource, url: feedback_path, builder: EtGdsDesignSystem.form_builder_class do |f|
+
       .form-fields
         = f.invisible_captcha :subtitle
-        = f.input :comments,
-          as: :text,
-          input_html: { rows: 5 }
-        = f.input :suggestions,
-          as: :text,
-          input_html: { rows: 5 }
-        = f.input :email_address
+        = f.text_area :comments, label: { text: "#{t('claims.feedback.feedback_comments')}"}, max_chars: 2500, rows: 5
```

Comment:
You dont need label: { text: <blablabla> }

You can just do label: <blablabla> when using the methods that are not prefixed with govuk_